source/multiple-mongosyncs.txt

```diff
@@ -133,6 +127,20 @@ To configure multiple ``mongosync`` instances:
            --cluster1 "mongodb://user:password@cluster1host:27500" \
            --id shard02 --port 27602
 
+      The connection strings for the :option:`--cluster0` and 
+      :option:`--cluster1` options should point to ``mongos`` instances.
+      In the example, they use the same ``mongos`` instance.
+
+      Each ``mongosync`` instance:
+
+      - Connects to a ``mongos`` instance in the source cluster.
+      - Connects to a ``mongos`` instance in the destination cluster.
```

Comment:
I would say "mongos instances" instead of "a mongos instance". It is fine specify connection strings that include multiple mongos instances.

Reply:
Fixed